bin/cml-runner.js

```diff
@@ -326,9 +351,15 @@ const opts = yargs
     'idle-timeout',
     'Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to 0 disables automatic shutdown'
   )
-  .default('name', RUNNER_NAME)
-  .describe('name', 'Name displayed in the repository once registered')
-
+  .default('name')
+  .describe('name', 'Name displayed in the repository once registered cml-{ID}')
+  .coerce('name', (val) => val || RUNNER_NAME)
```

Comment:
What if we use the idiomatic `default` to show users how the generaetd name will look like? It's a bit more involved, but might look nicer.

```suggestion
  .default('name', 'cml-{identifier}')
  .describe('name', 'Name displayed in the repository once registered')
  .coerce('name', (val) => val == 'cml-{identifier}' ? RUNNER_NAME : val)
```

#### Before

```console
  --name                       Name displayed in the repository once registered
                               cml-{ID}
```

#### After

```console
  --name                       Name displayed in the repository once registered
                                                  [default: "cml-{identifier}"]
```

Reply:
sounds good

Reply:
Probably almost every single user will understand ID (developers) but we can change it

Reply:
Warning: preference for unabbreviated words is one of the most salient traits of the [OCD](https://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder) I don't have; 😉  using `ID` instead is perfectly fine for me.

Reply:
I've edited the suggestion, feel free to commit it.